Misc/NEWS.d/next/Core and Builtins/2022-02-07-17-40-23.bpo-46671.CmpNlQ.rst

```diff
@@ -0,0 +1 @@
+Change exception message of :func:`min()` and :func:`max()` to refer to an iterable rather than a sequence if an empty iterable of values to select from was provided..
```

Comment:
```suggestion
Change exception message of :func:`min()` and :func:`max()` to refer to an iterable rather than a sequence if an empty iterable of values to select from was provided.
```